### DISCUSSION TITLE
DaskExecutor is removed in 2.8.1

### DISCUSSION THREAD

**Discussion by lutuantai95** [apache/airflow#36976] [Q&A] [answered]
Was DaskExecutor removed in 2.8.1 version?

**Comment by potiuk** [comment, marked as answer]
Yes and No. You should check release notes: 

https://airflow.apache.org/docs/apache-airflow/stable/release_notes.html#some-executors-have-been-moved-to-corresponding-providers-32767

Dask executor has been moved to provider in 2.7.0 https://airflow.apache.org/docs/apache-airflow-providers-daskexecutor/stable/index.html - there you need to install daskexecutor provider if you want to use the executor.

However in 2.8.0 daskexecutor has been marked as removed from maintenance - which means that we are no longer going to maintain it - other than bugfixes and you can continue using the released provider (last release was version 1.1.1). There will be no more releases ot the executor. It is also excluded from the reference image and you need to extend the image and add daskexecutor provider on your own if you want to continue to use dask executor.

This has been agreed in the devlist https://lists.apache.org/thread/fxv44cqqljrrhll3fdpdgc9h9fz5ghcy (you can find all the discussion links there and follow the) 

The executor uses stable Executor API that we implemented in Airflow in 2.7 https://cwiki.apache.org/confluence/display/AIRFLOW/AIP-51+Removing+Executor+Coupling+from+Core+Airflow so the idea is that you will be able to use the released daskecutor provider in perpetuity - i.e. it will not receive bugfixes nor new features (unless someone will fork it and release new versions of it - even you could take it over if you will want to add new features). 

But what you have should continue to work
